docs/kernel/devices.md

```diff
@@ -31,26 +96,91 @@ Partial list of IOCTL codes:
 | *SCE_HDMI_IOCTL_GET_HDMI_CONFIG*       | 0xC0108D10 |
 | *SCE_HDMI_IOCTL_GET_HDMI_STATE*        | 0xC0088D0C |
 | *SCE_HDMI_IOCTL_IC_INIT*               | 0x20008D01 |
+| *SCE_HDMI_IOCTL_ORBIS_AUDIO_UPDATE_TICK_PARAMS* | 0xC0045002 |
 | *SCE_HDMI_IOCTL_SET_GAMUTMETA_DATA*    | 0xC0108D07 |
+| *SCE_HDMI_IOCTL_SNDCTL_DSP_CHANNELS*   | 0xC0045002 |
+| *SCE_HDMI_IOCTL_SNDCTL_DSP_SETFMT*     | 0x40045004 |
+| *SCE_HDMI_IOCTL_SNDCTL_DSP_SETFRAGMENT* | 0x40045004 |
+| *SCE_HDMI_IOCTL_SNDCTL_DSP_SPEED*      | 0xC0045002 |
+| *SCE_HDMI_IOCTL_SNDCTL_DSP_SYNCGROUP*  | 0xC048501C |
 | *SCE_HDMI_IOCTL_VIDEO_CONFIG*          | 0xC0148D02 |
 
-
 ## `/dev/hmd_cmd`
 
 ### IOCTLs
-
 Partial list of IOCTL codes:
 
 | Name                                   | Code       |
 |----------------------------------------|------------|
 | *SCE_HMD_CMD_IOCTL_GET_MENU_SETTING*   | 0x8008A018 |
 | *SCE_HMD_CMD_IOCTL_SET_MENU_SETTING*   | 0x8001A02D |
+Seems involved in submitting PM4 command lists to the CP.
 
+## `/dev/mbus_kmod`?
+### IOCTLs
+Partial list of IOCTL codes:
 
-## `/dev/gc`
+| Name                                   | Code       |
+|----------------------------------------|------------|
+| *SCE_MBUS_KMOD_IOCTL_GET_DEVICE_INFO*  | 0x00000000 |
+| *SCE_MBUS_KMOD_IOCTL_GET_DEVICE_PROPERTY* | 0x00000000 |
+| *SCE_MBUS_KMOD_IOCTL_SET_SYSTEM_STATE* | 0x00000000 |
+| *SCE_MBUS_KMOD_IOCTL_SET_USB_POWER_BLACK_LIST* | 0x00000000 |
```

Comment:
It's quite unlikely that all these names were mapped to the IOCTL *0x00000000*. Was it impossible to find: Maybe mentioned in the strings only without the corresponding code? Or maybe the code passed to the `ioctl` call was hard to recover due to nasty compiler optimizations?

If so, simply replace the code by *???*, otherwise please just double-check it if you can.

Reply:
Oh yeah, 0 means not found. Forgot to replace them.